scripts/apps/search/components/fields/slugline.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {createMarkUp} from '../../helpers';
+
+export function slugline(props) {
+    if (props.item.slugline) {
+        return (
+            <span
+                key="slugline"
+                className="field--slugline"
+                dangerouslySetInnerHTML={createMarkUp(props.item.slugline)}
+                data-test-id="field--slugline"
+            />
+        );
+    } else {
+        return null;
+    }
+}
+
+slugline['propTypes'] = {
+    item: PropTypes.any,
```

Comment:
should this be an `PropTypes.object`?

Reply:
It should be `IArticle`. I've added interfaces to all field components.